cluster/gce/util.sh

```diff
@@ -3726,10 +3762,10 @@ function kube-down() {
 #
 # NOTE: Must be in sync with get-replica-name-regexp and set-replica-name.
 function get-replica-name() {
-  echo $(gcloud compute instances list \
+  gcloud compute instances list \
```

Comment:
this has the potential to change the exit code, are we sure we want to do that?

Reply:
Ah, good point. It does seem strange that a function should pretend to successfully return something correctly when it is failing, but the purpose of this patch was not to change the behavior. I will change this back to the echo and add a comment + a shellcheck disable.